Text/CustomEntitySearch/azuredeploy.json

```diff
@@ -0,0 +1,118 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourcePrefix": {
+      "type": "string",
+      "defaultValue": "psbes",
+      "metadata": {
+        "description": "Prefix for all resources created by this template"
+      }
+    },
+      "storageAccountType": {
+        "type": "string",
+        "defaultValue": "Standard_LRS",
+        "allowedValues": [
+          "Standard_LRS",
+          "Standard_GRS",
+          "Standard_ZRS",
+          "Premium_LRS"
+        ],
+        "metadata": {
+          "description": "Storage Account type"
+        }
+      }
+  },
+  "variables": {
+    "repoURL": "https://github.com/Azure-Samples/azure-search-power-skills",
+    "functionProject": "Text\\CustomEntitySearch\\CustomEntitySearch.csproj",
+    "websiteName": "[toLower(concat(parameters('resourcePrefix'), '-site-', uniqueString(resourceGroup().id)))]",
+    "storageAccount": "[toLower(concat(parameters('resourcePrefix'), uniqueString(resourceGroup().id)))]",
+    "functionAppName": "[toLower(concat(parameters('resourcePrefix'), '-function-app-', uniqueString(resourceGroup().id)))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-10-01",
+      "name": "[variables('storageAccount')]",
+      "type": "Microsoft.Storage/storageAccounts",
+      "location": "[resourceGroup().location]",
+      "sku": {
+        "name": "[parameters('storageAccountType')]"
+      },
+      "kind": "Storage"
+    },
+    {
+      "apiVersion": "2016-08-01",
+      "name": "[variables('functionAppName')]",
+      "type": "Microsoft.Web/sites",
+      "kind": "functionapp",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount'))]"
+      ],
+      "properties": {
+        "name": "[variables('functionAppName')]",
+        "kind": "functionapp",
+        "httpsOnly": true,
+        "siteConfig": {
+          "appSettings": [
+            {
+              "name": "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING",
+              "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(resourceId('CogSearch','Microsoft.Storage/storageAccounts', variables('storageAccount')), '2015-06-15').key1)]"
```

Comment:
For this setting and the next two, remove `"CogSearch",`. This is a mistake I made in my own deployment files that I fixed on Tuesday in all existing skills.